### DISCUSSION TITLE
Use ISBN as an alternative to reassociating read progress

### DISCUSSION THREAD

**Discussion by zachstultz** [gotson/komga#1553] [Ideas]
### Describe your suggested feature

In the event that a file's name and hash change completely, but the ISBN number is still the same, use that as an alternative to reassociate the read progress to the new file.

I know that not every file has an ISBN number, but I still think it would be a worthwhile addition.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by edgehub-cloud** [comment]
I support this is  isbn thing
